hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/IncrementalRelation.scala

```diff
@@ -121,6 +125,13 @@ class IncrementalRelation(val sqlContext: SQLContext,
 
   override def schema: StructType = usedSchema
 
+  private def isSchemaEvolutionEnabledOnRead = {
+    optParams.getOrElse(DataSourceReadOptions.SCHEMA_EVOLUTION_ENABLED.key,
```

Comment:
Let's abstract this config retrieval in a common method (we can reuse it in other places as well)